src/core/blueprint/operators/apollo_federation.rs

```diff
@@ -13,6 +15,22 @@ use crate::core::try_fold::TryFold;
 use crate::core::valid::{Valid, Validator};
 use crate::core::{config, Type};
 
+// ref: https://www.apollographql.com/docs/federation/federated-schemas/federated-directives
+const FEDERATION_DIRECTIVES: &[&str] = &[
```

Comment:
we have a similar list of federation directives here https://github.com/tailcallhq/tailcall/pull/2935/files#diff-f40f5478a0ee614381f25d37362c9c9c7779df174a375a166cf70e3e392e5a5dR109 can we reuse the definition?